### PR TITLE
De-fiboisation

### PR DESCRIPTION
**Pull request by mereolog**
Most references in bash code were removed.

### CODE REVIEW

**Review by trojczak — DISMISSED** [review]

**Review by przemekgradzki — CHANGES_REQUESTED** [review]
Versioned URLs are generated incorrectly, e.g.

[<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/BE/Trusts/Trusts/"/>](https://github.com/edmcouncil/fibo/blob/master/SEC/Funds/Funds.rdf#L77)

should be changed to:
`<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/master/latest/BE/Trusts/Trusts/"/>`

or
[<owl:versionIRI rdf:resource="https://spec.edmcouncil.org/fibo/ontology/SEC/20220101/Funds/Funds/"/>](https://github.com/edmcouncil/fibo/blob/master/SEC/Funds/Funds.rdf#L93)
should be changed to:
`<owl:versionIRI rdf:resource="https://spec.edmcouncil.org/fibo/ontology/master/latest/SEC/Funds/Funds/"/>`

**Review by rivettp — CHANGES_REQUESTED** [review]
I disagree with this change,  it could have widespread impact.  The URLs were carefully designed

**Comment by mereolog** [comment]
> I disagree with this change, it could have widespread impact. The URLs were carefully designed

I understand that you object to changes to FIBO IRIs.
We do not intend to change them - the problem reported by @przemekgradzki should be fixed in the last commit.

File: README.md [hunk 1]
```diff
@@ -1,19 +1,17 @@
-<img src="https://spec.edmcouncil.org/fibo/htmlpages/master/latest/img/logo.66a988fe.png" width="150" align="right"/>
+<img src="https://spec.edmcouncil.org/fibo/htmlpages/master/develop/img/logo.66a988fe.png" width="150" align="right"/>
```
Comment (przemekgradzki) [inline README.md]: Should be:
`https://spec.edmcouncil.org/fibo/htmlpages/develop/latest/img/logo.66a988fe.png`

File: publisher/product/ontology/build-about.sh [hunk 2]
```diff
@@ -33,7 +25,7 @@ function ontologyCreateAboutFiles () {
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> 
 @prefix owl: <http://www.w3.org/2002/07/owl#> 
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#>
-<${tag_root_url}/LoadFIBOProd> a owl:Ontology;
+<${tag_root_url}/Load${ONTPUB_FAMILY}Prod> a owl:Ontology;
```
Comment (przemekgradzki) [inline publisher/product/ontology/build-about.sh]: Should be uppercase:
`<${tag_root_url}/Load${ONTPUB_FAMILY^^}Prod> a owl:Ontology;`

File: publisher/product/ontology/build-about.sh [hunk 3]
```diff
@@ -48,11 +40,11 @@ __HERE__
     "${JENA_ARQ}" \
       --data="${tmpAboutFileProd}" \
       --query="${SCRIPT_DIR}/lib/echo.sparql" \
-      --results=RDF > "${tag_root}/LoadFIBOProd.rdf"
+      --results=RDF > "${tag_root}/Load${ONTPUB_FAMILY}Prod.rdf"
```
Comment (przemekgradzki) [inline publisher/product/ontology/build-about.sh]: Should be uppercase:
`--results=RDF > "${tag_root}/Load${ONTPUB_FAMILY^^}Prod.rdf"`

File: publisher/product/ontology/build-about.sh [hunk 4]
```diff
@@ -48,11 +40,11 @@ __HERE__
...
     # 2> "${TMPDIR}/err.tmp"
 
     if [ -s "${TMPDIR}/err.tmp" ] ; then
-      warning "no RDF XML output generated.  Use LoadFIBOProd.ttl file instead"
+      warning "no RDF XML output generated.  Use Load${ONTPUB_FAMILY}Prod.ttl file instead"
```
Comment (przemekgradzki) [inline publisher/product/ontology/build-about.sh]: Should be uppercase:
`warning "no RDF XML output generated. Use Load${ONTPUB_FAMILY^^}Prod.ttl file instead"`

File: publisher/product/ontology/build-about.sh [hunk 5]
```diff
@@ -65,7 +57,7 @@ __HERE__
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> 
 @prefix owl: <http://www.w3.org/2002/07/owl#> 
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#>
-<${tag_root_url}/LoadFIBODev> a owl:Ontology;
+<${tag_root_url}/Load${ONTPUB_FAMILY}Dev> a owl:Ontology;
```
Comment (przemekgradzki) [inline publisher/product/ontology/build-about.sh]: Should be uppercase:
`<${tag_root_url}/Load${ONTPUB_FAMILY^^}Dev> a owl:Ontology;`

File: publisher/product/ontology/build-about.sh [hunk 6]
```diff
@@ -86,10 +78,10 @@ __HERE__
     "${JENA_ARQ}" \
       --data="${tmpAboutFileDev}" \
       --query="${SCRIPT_DIR}/lib/echo.sparql" \
-      --results=RDF > "${tag_root}/LoadFIBODev.rdf" 2> "${TMPDIR}/err.tmp"
+      --results=RDF > "${tag_root}/Load${ONTPUB_FAMILY}Dev.rdf" 2> "${TMPDIR}/err.tmp"
```
Comment (przemekgradzki) [inline publisher/product/ontology/build-about.sh]: Should be uppercase:
`--results=RDF > "${tag_root}/Load${ONTPUB_FAMILY^^}Dev.rdf" 2> "${TMPDIR}/err.tmp"`

File: publisher/product/ontology/build-about.sh [hunk 7]
```diff
@@ -86,10 +78,10 @@ __HERE__
...
 
     if [ -s "${TMPDIR}/err.tmp" ] ; then
-      warning "no RDF XML output generated.  Use LoadFIBODev.ttl file instead"
+      warning "no RDF XML output generated.  Use Load${ONTPUB_FAMILY}Dev.ttl file instead"
```
Comment (przemekgradzki) [inline publisher/product/ontology/build-about.sh]: Should be uppercase:
`warning "no RDF XML output generated. Use Load${ONTPUB_FAMILY^^}Dev.ttl file instead"`

File: publisher/product/ontology/build-theallfile.sh [hunk 8]
```diff
@@ -32,7 +32,7 @@ function ontologyCreateTheAllTtlFile() {
       --dir="${INPUT}/LCC" \
       --dir=${tag_root} \
       --top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/AboutFIBODev/" \
```
Comment (przemekgradzki) [inline publisher/product/ontology/build-theallfile.sh]: Should be:
`--top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/About${ONTPUB_FAMILY^^}Dev/" \`

File: publisher/product/ontology/build-theallfile.sh [hunk 9]
```diff
@@ -32,7 +32,7 @@ function ontologyCreateTheAllTtlFile() {
...
-      --top="https://spec.edmcouncil.org/fibo/ontology/MetadataFIBO/" \
+      --top="https://spec.edmcouncil.org/${ONTPUB_FAMILY}/ontology/Metadata${ONTPUB_FAMILY}/" \
```
Comment (przemekgradzki) [inline publisher/product/ontology/build-theallfile.sh]: Should be uppercase:
`--top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/Metadata${ONTPUB_FAMILY^^}/" \`

File: publisher/product/ontology/build-theallfile.sh [hunk 10]
```diff
@@ -41,8 +41,8 @@ function ontologyCreateTheAllTtlFile() {
     python3 ${SCRIPT_DIR}/lib/trigify.py \
       --dir="${INPUT}/LCC" \
       --dir=${tag_root} \
-      --top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/AboutFIBODev/" \
-      --top="https://spec.edmcouncil.org/fibo/ontology/MetadataFIBO/" \
+      --top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/About${ONTPUB_FAMILY}Dev/" \
```
Comment (przemekgradzki) [inline publisher/product/ontology/build-theallfile.sh]: Should be uppercase:
`--top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/About${ONTPUB_FAMILY^^}Dev/" \`

File: publisher/product/ontology/build-theallfile.sh [hunk 11]
```diff
@@ -41,8 +41,8 @@ function ontologyCreateTheAllTtlFile() {
...
+      --top="https://spec.edmcouncil.org/${ONTPUB_FAMILY}/ontology/Metadata${ONTPUB_FAMILY}/" \
```
Comment (przemekgradzki) [inline publisher/product/ontology/build-theallfile.sh]: Should be uppercase:
`--top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/Metadata${ONTPUB_FAMILY^^}/" \`

File: publisher/product/ontology/build-theallfile.sh [hunk 12]
```diff
@@ -84,7 +85,7 @@ SELECT DISTINCT ?modfile WHERE {
     .
   }
   UNION {
-    BIND ("MetadataFIBO.rdf" AS ?modfile)
+    BIND ("Metadata${ONTPUB_FAMILY}.rdf" AS ?modfile)
```
Comment (przemekgradzki) [inline publisher/product/ontology/build-theallfile.sh]: Should be uppercase:
`BIND ("Metadata${ONTPUB_FAMILY^^}.rdf" AS ?modfile)`

File: publisher/product/ontology/build.sh [hunk 13]
```diff
@@ -523,26 +537,26 @@ function ontologyZipFiles () {
     find . -type f -name \*\.ttl -exec /bin/bash -c 'perl -pi -e "s/^\s*(\@prefix)\s+([^\s]+)\s+(\<[^\>]*\>)\s+(\.)\s*$/\1 \2 \3 \4\n/g" "{}"' \;
 
 
-    ${FIND}  "${family_product_branch_tag}" -name '*.ttl' -print | ${GREP} -v etc |  ${GREP} -v "LoadFIBOProd.ttl" | grep -v About |  xargs zip ${zipttlDevFile}
+    ${FIND}  "${family_product_branch_tag}" -name '*.ttl' -print | ${GREP} -v etc |  ${GREP} -v "Load${ONTPUB_FAMILY}Prod.ttl" | grep -v About |  xargs zip ${zipttlDevFile}
```
Comment (przemekgradzki) [inline publisher/product/ontology/build.sh]: Should be uppercase:
`${FIND} "${family_product_branch_tag}" -name '*.ttl' -print | ${GREP} -v etc | ${GREP} -v "Load${ONTPUB_FAMILY^^}Prod.ttl" | grep -v About | xargs zip ${zipttlDevFile}`

File: publisher/product/ontology/build.sh [hunk 14]
```diff
@@ -523,26 +537,26 @@ function ontologyZipFiles () {
...
     ${FIND}  "${family_product_branch_tag}" -name '*catalog*.xml' -print | xargs zip ${zipttlDevFile}
 
-    ${FIND}  "${family_product_branch_tag}" -name '*.rdf' -print | ${GREP} -v etc | ${GREP} -v "LoadFIBOProd.rdf" |  grep -v About |xargs zip ${ziprdfDevFile}
+    ${FIND}  "${family_product_branch_tag}" -name '*.rdf' -print | ${GREP} -v etc | ${GREP} -v "Load${ONTPUB_FAMILY}Prod.rdf" |  grep -v About |xargs zip ${ziprdfDevFile}
```
Comment (przemekgradzki) [inline publisher/product/ontology/build.sh]: Should be uppercase:
`${FIND} "${family_product_branch_tag}" -name '*.rdf' -print | ${GREP} -v etc | ${GREP} -v "Load${ONTPUB_FAMILY^^}Prod.rdf" | grep -v About |xargs zip ${ziprdfDevFile}`

File: publisher/product/ontology/build.sh [hunk 15]
```diff
@@ -523,26 +537,26 @@ function ontologyZipFiles () {
...
     ${FIND}  "${family_product_branch_tag}" -name '*catalog*.xml' -print | xargs zip ${ziprdfDevFile}
 
-    ${FIND}  "${family_product_branch_tag}" -name '*.jsonld' -print | ${GREP} -v etc | ${GREP} -v "LoadFIBOProd.jsonld" |  grep -v About |  xargs zip ${zipjsonldDevFile}
+    ${FIND}  "${family_product_branch_tag}" -name '*.jsonld' -print | ${GREP} -v etc | ${GREP} -v "Load${ONTPUB_FAMILY}Prod.jsonld" |  grep -v About |  xargs zip ${zipjsonldDevFile}
```
Comment (przemekgradzki) [inline publisher/product/ontology/build.sh]: Should be uppercase:
`${FIND} "${family_product_branch_tag}" -name '*.jsonld' -print | ${GREP} -v etc | ${GREP} -v "Load${ONTPUB_FAMILY^^}Prod.jsonld" | grep -v About | xargs zip ${zipjsonldDevFile}`

File: publisher/product/ontology/build.sh [hunk 16]
```diff
@@ -523,26 +537,26 @@ function ontologyZipFiles () {
...
     ${FIND}  "${family_product_branch_tag}" -name '*catalog*.xml' -print | xargs zip ${zipjsonldDevFile}
 
 
     
 
     ${GREP} -r 'utl-av[:;.]Release' "${family_product_branch_tag}" | ${GREP} -F ".ttl" | ${SED} 's/:.*$//' | xargs zip -r ${zipttlProdFile}
-    ${FIND}  "${family_product_branch_tag}" -name '*Load*.ttl' -print | ${GREP} -v "LoadFIBODev.ttl" |  xargs zip ${zipttlProdFile}
+    ${FIND}  "${family_product_branch_tag}" -name '*Load*.ttl' -print | ${GREP} -v "Load${ONTPUB_FAMILY}Dev.ttl" |  xargs zip ${zipttlProdFile}
```
Comment (przemekgradzki) [inline publisher/product/ontology/build.sh]: Should be uppercase:
`${FIND} "${family_product_branch_tag}" -name '*Load*.ttl' -print | ${GREP} -v "Load${ONTPUB_FAMILY^^}Dev.ttl" | xargs zip ${zipttlProdFile}`

File: publisher/product/ontology/build.sh [hunk 17]
```diff
@@ -523,26 +537,26 @@ function ontologyZipFiles () {
...
     ${FIND}  "${family_product_branch_tag}" -name '*catalog*.xml' -print | xargs zip ${zipttlProdFile}
     ${GREP} -r 'utl-av[:;.]Release' "${family_product_branch_tag}" | ${GREP} -F ".rdf" |   ${SED} 's/:.*$//' | xargs zip -r ${ziprdfProdFile}
-    ${FIND}  "${family_product_branch_tag}" -name '*Load*.rdf' -print | ${GREP} -v "LoadFIBODev.rdf" | xargs zip ${ziprdfProdFile}
+    ${FIND}  "${family_product_branch_tag}" -name '*Load*.rdf' -print | ${GREP} -v "Load${ONTPUB_FAMILY}Dev.rdf" | xargs zip ${ziprdfProdFile}
```
Comment (przemekgradzki) [inline publisher/product/ontology/build.sh]: Should be uppercase:
`${FIND} "${family_product_branch_tag}" -name '*Load*.rdf' -print | ${GREP} -v "Load${ONTPUB_FAMILY^^}Dev.rdf" | xargs zip ${ziprdfProdFile}`

File: publisher/product/ontology/build.sh [hunk 18]
```diff
@@ -523,26 +537,26 @@ function ontologyZipFiles () {
...
     ${FIND}  "${family_product_branch_tag}" -name '*catalog*.xml' -print | xargs zip ${ziprdfProdFile}
-    ${GREP} -r 'https://spec.edmcouncil.org/fibo/ontology/FND/Utilities/AnnotationVocabulary/Release' "${family_product_branch_tag}" | ${GREP} -F ".jsonld" |   ${SED} 's/:.*$//' | xargs zip -r ${zipjsonldProdFile}
-    ${FIND}  "${family_product_branch_tag}" -name '*Load*.jsonld' -print | ${GREP} -v "LoadFIBODev.jsonld" | xargs zip ${zipjsonldProdFile}
+    ${GREP} -r 'https://spec.edmcouncil.org/${ONTPUB_FAMILY}/ontology/FND/Utilities/AnnotationVocabulary/Release' "${family_product_branch_tag}" | ${GREP} -F ".jsonld" |   ${SED} 's/:.*$//' | xargs zip -r ${zipjsonldProdFile}
+    ${FIND}  "${family_product_branch_tag}" -name '*Load*.jsonld' -print | ${GREP} -v "Load${ONTPUB_FAMILY}Dev.jsonld" | xargs zip ${zipjsonldProdFile}
```
Comment (przemekgradzki) [inline publisher/product/ontology/build.sh]: Should be uppercase:
`${FIND} "${family_product_branch_tag}" -name '*Load*.jsonld' -print | ${GREP} -v "Load${ONTPUB_FAMILY^^}Dev.jsonld" | xargs zip ${zipjsonldProdFile}`

File: publisher/product/ontology/build.sh [hunk 19]
```diff
@@ -523,26 +537,26 @@ function ontologyZipFiles () {
...
     ${GREP} -r 'utl-av[:;.]Release' "${family_product_branch_tag}" | ${GREP} -F ".rdf" |   ${SED} 's/:.*$//' | xargs zip -r ${ziprdfProdFile}
-    ${FIND}  "${family_product_branch_tag}" -name '*Load*.rdf' -print | ${GREP} -v "LoadFIBODev.rdf" | xargs zip ${ziprdfProdFile}
+    ${FIND}  "${family_product_branch_tag}" -name '*Load*.rdf' -print | ${GREP} -v "Load${ONTPUB_FAMILY}Dev.rdf" | xargs zip ${ziprdfProdFile}
     ${FIND}  "${family_product_branch_tag}" -name '*catalog*.xml' -print | xargs zip ${ziprdfProdFile}
-    ${GREP} -r 'https://spec.edmcouncil.org/fibo/ontology/FND/Utilities/AnnotationVocabulary/Release' "${family_product_branch_tag}" | ${GREP} -F ".jsonld" |   ${SED} 's/:.*$//' | xargs zip -r ${zipjsonldProdFile}
-    ${FIND}  "${family_product_branch_tag}" -name '*Load*.jsonld' -print | ${GREP} -v "LoadFIBODev.jsonld" | xargs zip ${zipjsonldProdFile}
+    ${GREP} -r 'https://spec.edmcouncil.org/${ONTPUB_FAMILY}/ontology/FND/Utilities/AnnotationVocabulary/Release' "${family_product_branch_tag}" | ${GREP} -F ".jsonld" |   ${SED} 's/:.*$//' | xargs zip -r ${zipjsonldProdFile}
```
Comment (przemekgradzki) [inline publisher/product/ontology/build.sh]: The entire URL

_https://spec.edmcouncil.org/${ONTPUB_FAMILY}/ontology/FND/Utilities/AnnotationVocabulary/Release_

is specific to FIBO.

File: publisher/product/ontology/build.sh [hunk 20]
```diff
@@ -677,41 +691,41 @@ __HERE__
 
 
   log "Getting metadata"
-  "${JENA_ARQ}" $(find "${source_family_root}" -name "MetadataFIBO.rdf" | grep -v "/etc/" | sed "s/^/--data=/") \
+  "${JENA_ARQ}" $(find "${source_family_root}" -name "Metadata${ONTPUB_FAMILY}.rdf" | grep -v "/etc/" | sed "s/^/--data=/") \
```
Comment (przemekgradzki) [inline publisher/product/ontology/build.sh]: Should be uppercase:
`"${JENA_ARQ}" $(find "${source_family_root}" -name "Metadata${ONTPUB_FAMILY^^}.rdf" | grep -v "/etc/" | sed "s/^/--data=/") \`

File: publisher/product/ontology/build.sh (hunk 19 above)
Comment (mereolog) [reply publisher/product/ontology/build.sh]: For now this must stay as it is.

File: publisher/product/ontology/build.sh [hunk 21]
```diff
@@ -656,7 +670,7 @@ __HERE__
 
 	  ${FIND} . -mindepth 2 -name '*.ttl' -print | while read file; do quadify "$file"; done > "${DevQuadsFile}"
 	  echo "starting prod"
-	  ${GREP} -rl 'fibo-fnd-utl-av:hasMaturityLevel fibo-fnd-utl-av:Release' | \
+	  ${GREP} -rl '${ONTPUB_FAMILY}-fnd-utl-av:hasMaturityLevel ${ONTPUB_FAMILY}-fnd-utl-av:Release' | \
```
Comment (przemekgradzki) [inline publisher/product/ontology/build.sh]: The `-fnd-utl-av:hasMaturityLevel` and `-fnd-utl-av:Release` fragments are specific to FIBO.

File: publisher/publish.sh [hunk 22]
```diff
@@ -132,18 +131,18 @@ function zipOntologyFiles () {
     # Make sure that everything is world readable before we zip it
     #
     chmod -R g+r,o+r .
-    zip -r ${zipttlDevFile} "${family_product_branch_tag}" -x \*.rdf \*.zip  \*.jsonld \*AboutFIBOProd.ttl
-    zip -r ${ziprdfDevFile} "${family_product_branch_tag}" -x \*.ttl \*.zip \*.jsonld \*AboutFIBOProd.rdf
-    zip -r ${zipjsonldDevFile} "${family_product_branch_tag}" -x \*.ttl \*.zip \*.rdf \*AboutFIBOProd.jsonld
+    zip -r ${zipttlDevFile} "${family_product_branch_tag}" -x \*.rdf \*.zip  \*.jsonld \*About${ONTPUB_FAMILY}Prod.ttl
```
Comment (przemekgradzki) [inline publisher/publish.sh]: Should be uppercase:
`zip -r ${zipttlDevFile} "${family_product_branch_tag}" -x \*.rdf \*.zip \*.jsonld \*About${ONTPUB_FAMILY^^}Prod.ttl`

File: publisher/publish.sh [hunk 23]
```diff
@@ -132,18 +131,18 @@ function zipOntologyFiles () {
...
+    zip -r ${ziprdfDevFile} "${family_product_branch_tag}" -x \*.ttl \*.zip \*.jsonld \*About${ONTPUB_FAMILY}Prod.rdf
```
Comment (przemekgradzki) [inline publisher/publish.sh]: Should be uppercase:
`zip -r ${ziprdfDevFile} "${family_product_branch_tag}" -x \*.ttl \*.zip \*.jsonld \*About${ONTPUB_FAMILY^^}Prod.rdf`

File: publisher/publish.sh [hunk 24]
```diff
@@ -132,18 +131,18 @@ function zipOntologyFiles () {
...
+    zip -r ${zipjsonldDevFile} "${family_product_branch_tag}" -x \*.ttl \*.zip \*.rdf \*About${ONTPUB_FAMILY}Prod.jsonld
```
Comment (przemekgradzki) [inline publisher/publish.sh]: Should be uppercase:
`zip -r ${zipjsonldDevFile} "${family_product_branch_tag}" -x \*.ttl \*.zip \*.rdf \*About${ONTPUB_FAMILY^^}Prod.jsonld`

File: publisher/publish.sh [hunk 25]
```diff
@@ -132,18 +131,18 @@ function zipOntologyFiles () {
...
 
     ${GREP} -r 'utl-av[:;.]Release' "${family_product_branch_tag}" | ${GREP} -F ".ttl" | ${SED} 's/:.*$//' | xargs zip -r ${zipttlProdFile}
-    ${FIND}  "${family_product_branch_tag}" -name '*About*.ttl' -print | ${GREP} -v "AboutFIBODev.ttl" |  xargs zip ${zipttlProdFile}
+    ${FIND}  "${family_product_branch_tag}" -name '*About*.ttl' -print | ${GREP} -v "About${ONTPUB_FAMILY}Dev.ttl" |  xargs zip ${zipttlProdFile}
```
Comment (przemekgradzki) [inline publisher/publish.sh]: Should be uppercase:
`${FIND} "${family_product_branch_tag}" -name '*About*.ttl' -print | ${GREP} -v "About${ONTPUB_FAMILY^^}Dev.ttl" | xargs zip ${zipttlProdFile}`

File: publisher/publish.sh [hunk 26]
```diff
@@ -132,18 +131,18 @@ function zipOntologyFiles () {
...
     ${FIND}  "${family_product_branch_tag}" -name '*catalog*.xml' -print | xargs zip ${zipttlProdFile}
     ${GREP} -r 'utl-av[:;.]Release' "${family_product_branch_tag}" | ${GREP} -F ".rdf" |   ${SED} 's/:.*$//' | xargs zip -r ${ziprdfProdFile}
-    ${FIND}  "${family_product_branch_tag}" -name '*About*.rdf' -print | ${GREP} -v "AboutFIBODev.rdf" | xargs zip ${ziprdfProdFile}
+    ${FIND}  "${family_product_branch_tag}" -name '*About*.rdf' -print | ${GREP} -v "About${ONTPUB_FAMILY}Dev.rdf" | xargs zip ${ziprdfProdFile}
```
Comment (przemekgradzki) [inline publisher/publish.sh]: Should be uppercase:
`${FIND} "${family_product_branch_tag}" -name '*About*.rdf' -print | ${GREP} -v "About${ONTPUB_FAMILY^^}Dev.rdf" | xargs zip ${ziprdfProdFile}`

File: publisher/publish.sh [hunk 27]
```diff
@@ -132,18 +131,18 @@ function zipOntologyFiles () {
...
     ${FIND}  "${family_product_branch_tag}" -name '*catalog*.xml' -print | xargs zip ${ziprdfProdFile}
     ${GREP} -r 'utl-av[:;.]Release' "${family_product_branch_tag}" | ${GREP} -F ".jsonld" |   ${SED} 's/:.*$//' | xargs zip -r ${zipjsonldProdFile}
-    ${FIND}  "${family_product_branch_tag}" -name '*About*.jsonld' -print | ${GREP} -v "AboutFIBODev.jsonld" | xargs zip ${zipjsonldProdFile}
+    ${FIND}  "${family_product_branch_tag}" -name '*About*.jsonld' -print | ${GREP} -v "About${ONTPUB_FAMILY}Dev.jsonld" | xargs zip ${zipjsonldProdFile}
```
Comment (przemekgradzki) [inline publisher/publish.sh]: Should be uppercase:
`${FIND} "${family_product_branch_tag}" -name '*About*.jsonld' -print | ${GREP} -v "About${ONTPUB_FAMILY^^}Dev.jsonld" | xargs zip ${zipjsonldProdFile}`

File: publisher/product/ontology/build.sh (hunk 21 above)
Comment (mereolog) [reply publisher/product/ontology/build.sh]: See above.

File: publisher/product/ontology/build-theallfile.sh [hunk 28]
```diff
@@ -31,8 +31,8 @@ function ontologyCreateTheAllTtlFile() {
     python3 ${SCRIPT_DIR}/lib/trigify.py \
       --dir="${INPUT}/LCC" \
       --dir=${tag_root} \
-      --top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/AboutFIBODev/" \
-      --top="https://spec.edmcouncil.org/fibo/ontology/MetadataFIBO/" \
+      --top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/About${ONTPUB_FAMILY^^}Dev/" \
+      --top="https://spec.edmcouncil.org/${ONTPUB_FAMILY^^}/ontology/Metadata${ONTPUB_FAMILY^^}/" \
```
Comment (przemekgradzki) [inline publisher/product/ontology/build-theallfile.sh]: Should be:
`--top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/Metadata${ONTPUB_FAMILY^^}/" \`

File: publisher/product/ontology/build-theallfile.sh [hunk 29]
```diff
@@ -41,8 +41,8 @@ function ontologyCreateTheAllTtlFile() {
...
```
Comment (przemekgradzki) [inline publisher/product/ontology/build-theallfile.sh]: Should be:
`--top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/Metadata${ONTPUB_FAMILY^^}/" \`

**Review by przemekgradzki — APPROVED** [review]

**Pull request merged**